tests/unit/s2n_build_test.c

```diff
@@ -131,6 +158,15 @@ int main()
         }
     };
 
+    /* Ensure we are testing with supported libcryto variants.
+     *
+     * We might need to update s2n_libcrypto_is_openssl() when adding support
+     * for a new libcrypto.
```

Comment:
As discussed offline: I'd remove the "might". Even though updating s2n_libcrypto_is_openssl might not be the answer, it probably is and the reader should assume it is.